logging/bin/deploy_elasticsearch_open.sh

```diff
@@ -0,0 +1,291 @@
+   # K8s objects to ensure proper labels are used in case user chooses to configure additional single-role nodes
+   log_debug "Patching deployment [v4m-es-client]"
+   kubectl -n $LOG_NS patch deployment v4m-es-client --type=json --patch '[{"op": "add","path": "/spec/template/metadata/labels/esclient","value": "true" }]'
+
+   log_debug "Patching statefulset [v4m-es-data]"
+   kubectl -n $LOG_NS patch statefulset v4m-es-data  --type=json --patch '[{"op": "add","path": "/spec/template/metadata/labels/esdata","value": "true" }]'
+
+   # patching 'client' and 'data' _services_ to use new multi-role labels for node selection
+   log_debug "Patching  service [v4m-es-client-service]"
+   kubectl -n $LOG_NS patch service v4m-es-client-service --type=json --patch '[{"op": "remove","path": "/spec/selector/role"},{"op": "add","path": "/spec/selector/esclient","value": "true" }]'
+
+   log_debug "Patching  service [v4m-es-data-service]"
+   kubectl -n $LOG_NS patch service v4m-es-data-svc --type=json --patch '[{"op": "remove","path": "/spec/selector/role"},{"op": "add","path": "/spec/selector/esdata","value": "true" }]'
+else
+   log_debug "**********************************>Multirole Flag: $ES_MULTIROLE_NODES"
+fi
+
+# wait for pod to come up
+log_info "Waiting [90] seconds to allow PVCs for pod [v4m-es-master-0] to be matched with available PVs [$(date)]"
+sleep 90s
+
+# Confirm PVC is "bound" (matched) to PV
+pvc_status=$(kubectl -n $LOG_NS get pvc  data-v4m-es-master-0  -o=jsonpath="{.status.phase}")
+if [ "$pvc_status" != "Bound" ];  then
+      log_error "It appears that the PVC [data-v4m-es-master-0] associated with the [v4m-es-master-0] node has not been bound to a PV."
+      log_error "The status of the PVC is [$pvc_status]"
+      log_error "After ensuring all claims shown as Pending can be satisfied; run the remove_elasticsearch_open.sh script and try again."
+      exit 1
+fi
+log_info "The PVC [data-v4m-es-master-0] have been bound to PVs"
+
+# Need to wait 2-3 minutes for the elasticsearch to come up and running
+log_info "Checking on status of Elasticsearch pod before configuring [$(date)]"
+podready="FALSE"
+
+for pause in 40 30 20 15 10 10 10 15 15 15 30 30 30 30
+do
+   if [[ "$( kubectl -n $LOG_NS get pod v4m-es-master-0 -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}')" == *"True"* ]]; then
+      log_info "Pod [v4m-es-master-0] is ready...continuing."
+      podready="TRUE"
+      break
+   else
+      log_info "Pod  [v4m-es-master-0] is not ready yet...sleeping for [$pause] more seconds before checking again."
+      sleep ${pause}s
+   fi
+done
+
+if [ "$podready" != "TRUE" ]; then
+   log_error "The Elasticsearch pod [v4m-es-master-0] has NOT reached [Ready] status in the expected time; exiting."
+   log_error "Review pod's events and log to identify the issue and resolve it; run the remove_logging.sh script and try again."
+   exit 1
+fi
+
+# TO DO: Convert to curl command to detect ES is up?
```

Comment:
This is fine for now. Maybe we move away from fixed sleeps in the future to speed things up in the future as long as we have valid gate checks.